### PR TITLE
readthedocs updates

### PR DESCRIPTION
**Pull request by snafees**
Updating readthedocs to automatically take in the github readme, changed sphinx theme and added better formatting to the readthedocs page. Look at updated readthedocs here https://ortho-seqs.readthedocs.io/en/updating_readme/index.html# 

Many thanks to contributing to snafees/ortho_seqs!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] Ensure the test suite passes with [pytest](https://docs.pytest.org/en/latest/) . (command to run: `pytest` or `make coverage` if you want to see which lines don't have tests yet)
 - [x] Make sure your code is linted and autoformatted using [black](https://github.com/psf/black) (`black . --check`).
 - [x] Documentation in `usage.md` is updated
 - [x] `README.md` is updated

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
     description="A PyPI package to compute multivariate tensor-based orthogonal polynomials for sequence data and map phenotypes onto sequence space.",
-    name="ortho_seq_code",
+    name="ortho_seqs",
     version="1.0.1",
     url="https://github.com/snafees/ortho_seqs",
     packages=find_packages(),
     keywords=[
         "regression",
-        "covariance",
+        "projections" "covariance",
```
Comment (neevor) [inline setup.py:16]: I think a comma is missing.

File: setup.py [hunk 2]
```diff
@@ -7,16 +7,16 @@
...
         "rna",
         "dna",
-        "orthogonal polynomials",
+        "sequence-phenotype" "orthogonal polynomials",
```
Comment (neevor) [inline setup.py:19]: I think a comma is missing.

**Review by neevor — COMMENTED** [review]
I haven't done anything with pyqt in a loooong time so I only did a quick look over that part but it seems ok and is still in development so pretty low risk.

File: setup.py (hunk 1 above)
Comment (snafees) [reply setup.py]: omg, thank you!

File: setup.py (hunk 2 above)
Comment (snafees) [reply setup.py]: Thank you!

**Pull request merged**